motion_config.html

```diff
@@ -1676,6 +1682,7 @@ <h2><a name="Configuration_OptionsTopic"></a>  Configuration Options-Listed by T
             </tr>
             <tr>
               <td bgcolor="#edf4f9" ><a href="#stream_auth_method" >stream_auth_method</a> </td>
+              <td bgcolor="#edf4f9" ><a href="#stream_cors_header" >stream_cors_header</a> </td>
```

Comment:
Can we shuffle the lines down or just add this in the last section?  This is a table with four columns.